doc/Sharding.md

```diff
@@ -31,8 +31,9 @@ Vitess supports the following types of sharding operations:
 
 With these features, you can start with a single keyspace that contains
 all of your data (in multiple tables). As your database grows, you can
-move tables to different keyspaces and shard some or all of those keyspaces
-without any real downtime for your application.
+move tables to different keyspaces (vertical sharding) and shard some or
```

Comment:
I suggest to write `vertical split` instead of `vertical sharding` because that's the term we use throughout the code base.